### PR TITLE
Bonnie prior import export rake tasks

### PR DESCRIPTION
**Pull request by lukoktonos**
Moving QDM-based patient export/import to bonnie-prior as part of cql testing overhaul.  Removing QDM from master branch and leaving CQL only there.

Additionally added load_tasks to test_helper.rb and removed it from other places to avoid duplicated loading.

see https://github.com/projecttacoma/bonnie/pull/955

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1255
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered. 

Branch | Coverage Report
-- | --
bonnie-prior | 1499 / 2630 LOC (57.0%) covered.
bonnie-prior-import-export-tasks | 1557 / 2672 LOC (58.27%) covered.

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @jbradl11 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: test/unit/prune_dumps_test.rb [hunk 1]
```diff
@@ -17,7 +17,6 @@ def create_mock_dump(date)
       Dir.mkdir(DUMPS_FOLDER)
     end
     Dir.glob(File.join(DUMPS_FOLDER, '*.tgz')).each { |f| File.delete(f) }
-    Bonnie::Application.load_tasks
```
Comment (mayerm94) [inline test/unit/prune_dumps_test.rb:20]: What was the reason for this deletion?

Comment (lukoktonos) [reply test/unit/prune_dumps_test.rb:20]: This was causing rake tasks to run repeatedly in tests (+1 repetition each time this `load_tasks` method was called. This was moved into test_helper, which is required by this file as well as guarded with a check to see if `Rake.Tasks` is empty.

**Review by jbradl11 — APPROVED** [review]
Tests pass. Code looks good.

**Review by mayerm94 — APPROVED** [review]

**Comment by edeyoung** [comment]
failing travis due to known nokogiri issue that we are working on resolving.

**Pull request merged**